lib/cocoapods/target/pod_target.rb

```diff
@@ -356,55 +346,50 @@ def resources_bundle_target_label(bundle_name)
       "#{label}-#{bundle_name}"
     end
 
-    # @param  [Symbol] test_type
-    #         The test type to use for producing the test label.
+    # @param  [Specification] test_spec
+    #         The test spec to use for producing the test label.
     #
     # @return [String] The derived name of the test target.
     #
-    def test_target_label(test_type)
-      "#{label}-#{test_type.capitalize}-Tests"
+    def test_target_label(test_spec)
+      # TODO: do not use base_name here
```

Comment:
TODO

Reply:
What is the best thing to use here? is `base_name` uniq enough?